Hacktoberfest_2022_contributors_guidelines.md

```diff
@@ -0,0 +1,15 @@
+# Hacktoberfest 2022 contributors guidelines
+
+Just as we’re getting ready for the Hacktoberfest 2022 hype, allow us to remind you of a few guidelines to follow to make sure the event goes smoothly for everyone. These guidelines are there to support the already existing ones available on Hacktoberfest's [](https://hacktoberfest.digitalocean.com/resources)[official website](https://hacktoberfest.com). 
+
+📜 Hacktoberfest is a highly intense time for both organisers and participants. To optimise time and ensure we can get to your PR in the best possible delays, we won’t focus on the PR that don’t meet the quality standards shared on this year’s Hacktoberfest [website](https://hacktoberfest.com/participation/#spam). Indeed, the significant number of entries will force us to prioritise our tasks as we hope to review as much of our contributors’ hard work as possible.
+
+🔍 In addition to that, and to respect our team's work and the time they invest in reviewing your contributions, please make sure to incorporate all the feedback they have already offered before asking for additional feedback. If any of the comments they shared are unclear to you, please feel free to ask for clarification. Still, our team won't review again a PR that didn't take the globality of the initial feedback into account during the revision.
+
+⏰ We couldn’t be more pleased to see a high number of contributions! We kindly request your patience with us; we will focus on reviewing your PRs from Monday to Thursday, during usual working hours, CEST time. If you’re outside that timeframe, please don’t panic; we will get to yours soon.
+
+👑 October is a special time to celebrate the open source community and its contributors; coding contributions are more than welcome. Non-coding contributions will be right at home in the [Awesome Meilisearch](https://github.com/meilisearch/awesome-meilisearch) repo. Documentation PRs will not be opened this time around, as we feel like Hacktoberfest might not be the most relevant time of the year for them.
```

Comment:
Happy to add a link for the coding contribution, do you reckon if this one might be the most relevant: https://github.com/orgs/meilisearch/repositories

Reply:
I wanted to emphasize on `Awesome Meilisearch` separately because it's not really straightforward to add non-code contributions during Hacktoberfest. It's a good thing to help potential contributors understand what would be the process for them to qualify to Hacktoberfest.

And I found the query that Maryam was talking about: https://github.com/search?q=org%3Ameilisearch+topic%3Ahacktoberfest&type=Repositories

If we want to do the same for issues: https://github.com/search?q=org%3Ameilisearch+label%3Ahacktoberfest&type=Issues (we may want to link to them, there's 188 already of them)

Reply:
Do you think we could put both or it would create redundancy of information? If we write "here's the list of all our repositories accepting contributions and here you can find all the issues eligible for the event" (+links)?

Reply:
I don’t think it would be redundant, we can use both

Reply:
fantastic so I'll include them both then thanks for the feedback!